framework/source/class/qx/test/log/Filters.js

```diff
@@ -0,0 +1,64 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2009 1&1 Internet AG, Germany, http://www.1und1.de
+
+   License:
+     LGPL: http://www.gnu.org/licenses/lgpl.html
+     EPL: http://www.eclipse.org/org/documents/epl-v10.php
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+ * Fabian Jakobs (fjakobs)
+
```

Comment:
Hi all, of course you should take care not to alter the license terms of existing files. That's to be done in the proposed, official license change. For newly added files on the other hand, I don't see a problem to use the liberate MIT license, but doubt it'll make much of a difference to continuing to go with the existing dual license for now.